packages/neo-one-node-bin/src/bin/neo-one-node.ts

```diff
@@ -1,6 +1,13 @@
 // tslint:disable-next-line:no-implicit-dependencies
-import { startNode } from '@neo-one/node-bin';
+import { getDefaultConfig, startNode } from '@neo-one/node-bin';
+import rc from 'rc';
 
-startNode().catch(() => {
+const config = rc('neo-one-node', getDefaultConfig());
```

Comment:
This should watch the found files for config changes and create an observable of options.